app/presenters/export/csv/school_presenter_module.rb

```diff
@@ -94,4 +94,40 @@ def school_sharepoint_folder
   end
 
   alias_method :school_sharepoint_link_with_academy_label, :school_sharepoint_folder
+
+  def school_main_contact_name
```

Comment:
I think these would be way more efficient if we do all the work to get the contact once and the present the name, email and title from that.